### PR TITLE
feat: Update CG to use Oclif core and space for the `topicSeparator`

### PR DESCRIPTION
**Pull request by m-pizarro**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,9 +18,8 @@
     "@cloudgraph/sdk": "^0.14.2",
     "@graphql-tools/load-files": "^6.3.2",
     "@graphql-tools/merge": "^8.2.0",
-    "@oclif/command": "^1",
-    "@oclif/config": "^1",
-    "@oclif/plugin-help": "3.2.17",
+    "@oclif/core": "^1.6.0",
+    "@oclif/plugin-help": "^5.1.12",
     "@tiagonapoli/oclif-plugin-spaced-commands": "^1.0.1",
```
Comment (tyler-dunkel) [inline package.json:23]: This package can be removed I think

File: package.json [hunk 2]
```diff
@@ -41,15 +40,14 @@
     "live-plugin-manager": "0.15.1",
```
Comment (tyler-dunkel) [inline package.json:40]: I think we can maybe remove this package as well. (verify this first)

File: package.json [hunk 3]
```diff
@@ -18,9 +18,8 @@
     "@cloudgraph/sdk": "^0.14.2",
```
Comment (tyler-dunkel) [inline package.json:18]: Lets bump sdk to latest in this PR

**Review by tyler-dunkel — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (m-pizarro) [reply package.json:23]: Yes, I forgot to remove this...

File: package.json (hunk 2 above)
Comment (m-pizarro) [reply package.json:40]: That package is being used, for example here:

image

Comment (tyler-dunkel) [reply package.json:40]: I am referring to the `live-plugin-manager` not the `semver` package.

Comment (m-pizarro) [reply package.json:39]: Yes, uninstalling the `live-plugin-manager` package I had those errors. But adding semver package in the devDependencies now is working wtitouth it.

Comment (tyler-dunkel) [reply package.json]: hmm that is interesting, as`semver` is already a dep on the `dependencies` I think..

**Review by tyler-dunkel — APPROVED** [review]

**Pull request merged**